Provider/ImageProvider.php

```diff
@@ -174,6 +174,10 @@ public function generatePublicUrl(MediaInterface $media, $format)
      */
     public function generatePrivateUrl(MediaInterface $media, $format)
     {
+        if ($format == 'reference') {
```

Comment:
> Yoda condition because it's a current Symfony's convention

@phansys I don't think so. Or I missed something. Could you please give your source?

Nothing about Yoda condition here: http://symfony.com/doc/current/contributing/code/standards.html